app/src/test/java/com/github/polybooks/core/databaseImpl/OLBookDBTests.kt

```diff
@@ -150,9 +144,9 @@ class OLBookDBTests {
         assertNotNull(book.authors)
         assertEquals("paperback", book.format)
         assertNotNull(book.publishDate)
-        assertEquals(6,book.publishDate!!.month)
-        assertEquals(2020-1900,book.publishDate!!.year)
-        assertEquals(3,book.publishDate!!.date)
+        //This hurts, but otherwise we have problems with CEST and CET
```

Comment:
What hurts? Are you talking about the space after 2020 and not before -1900? btw can you add space after the dash?

Reply:
No the fact that 3 deprecated methods are being used XD 
I'm not that picky on spacing